modules/test/playwright/pages/portal-workflow-task-web/WorkflowTaskDetailsPage.ts

```diff
@@ -57,6 +58,7 @@ export class WorkflowTaskDetailsPage {
 		this.reviewActionMenu = page.locator(
 			'[id="_com_liferay_portal_workflow_task_web_portlet_MyWorkflowTaskPortlet_kldx___menu"]'
 		);
+		this.editAssetButton = page.locator('[title="Edit"]');
```

Comment:
can you please order this? it's bellow the doneButton like you did when declaring the locator itself